yb-voyager/cmd/assessMigrationCommand.go

```diff
@@ -156,87 +185,68 @@ func packAndSendAssessMigrationPayload(status string, errMsg string) {
+	}
+
+	var obfuscatedIssues []callhome.AssessmentIssueCallhome
+	for _, issue := range assessmentReport.Issues {
+		obfuscatedIssue := callhome.AssessmentIssueCallhome{
+			Category:              issue.Category,
+			CategoryDescription:   issue.CategoryDescription,
+			Type:                  issue.Type,
+			Name:                  issue.Name,
+			Description:           issue.Description,
+			Impact:                issue.Impact,
+			ObjectType:            issue.ObjectType,
+			ObjectName:            constants.OBFUSCATE_STRING,
+			SqlStatement:          constants.OBFUSCATE_STRING, // TODO(future): we can obfuscate sensitive info in SQL statement
+			DocsLink:              issue.DocsLink,
+			MinimumVersionFixedIn: issue.MinimumVersionFixedIn,
+		}
 
-	groupedByConstructType := lo.GroupBy(assessmentReport.UnsupportedQueryConstructs, func(q utils.UnsupportedQueryConstruct) string {
-		return q.ConstructTypeName
-	})
-	countByConstructType := lo.MapValues(groupedByConstructType, func(constructs []utils.UnsupportedQueryConstruct, _ string) int {
-		return len(constructs)
-	})
+		// allowing object name for unsupported extension issue type
+		if issue.Type == UNSUPPORTED_EXTENSION_ISSUE_TYPE {
+			// object name(i.e. extension name here) might be qualified with schema so just taking the last part
+			obfuscatedIssue.ObjectName = strings.Split(issue.ObjectName, ".")[len(strings.Split(issue.ObjectName, "."))-1]
+		}
 
-	var unsupportedFeatures []callhome.UnsupportedFeature
-	for _, feature := range assessmentReport.UnsupportedFeatures {
-		if feature.FeatureName == EXTENSION_FEATURE {
-			// For extensions, we need to send the extension name in the feature name
-			// for better categorization in callhome
-			for _, object := range feature.Objects {
-				unsupportedFeatures = append(unsupportedFeatures, callhome.UnsupportedFeature{
-					FeatureName:      fmt.Sprintf("%s - %s", feature.FeatureName, object.ObjectName),
-					ObjectCount:      1,
-					TotalOccurrences: 1,
-				})
-			}
-		} else {
-			var objects []string
-			if slices.Contains(featuresToSendObjectsToCallhome, feature.FeatureName) {
-				objects = lo.Map(feature.Objects, func(o ObjectInfo, _ int) string {
-					return o.ObjectName
-				})
+		for _, sensitiveDescription := range descriptionsIncludingSensitiveInformationToCallhome {
+			if sensitiveDescription == UNSUPPORTED_PG_SYNTAX_ISSUE_REASON && strings.HasPrefix(obfuscatedIssue.Description, sensitiveDescription) {
+				obfuscatedIssue.Description = sensitiveDescription
+			} else {
+				match, err := utils.MatchesFormatString(sensitiveDescription, obfuscatedIssue.Description)
+				if match {
+					obfuscatedIssue.Description, err = utils.ObfuscateFormatDetails(sensitiveDescription, obfuscatedIssue.Description, constants.OBFUSCATE_STRING)
+				}
+				if err != nil {
+					log.Errorf("error while matching issue description with sensitive descriptions: %v", err)
+					obfuscatedIssue.Description = constants.OBFUSCATE_STRING
+				}
```

Comment:
I think we don't need to send the description at all in the assessment issues as the `issue.Name/issue.Type` should be sufficient but in the cases this is not sufficient like `index on complex datatypes, etc..` we should not worry about sending this currently as we were not sending this before as well and we should separate out issues to have `issue.Type/Name` differentiate that.
cc: @makalaaneesh

Reply:
we did this for analyze as we don't want to regress there with less information but I think here we can go without descriptions as well.

Reply:
only problem is we won't know have the info like index access method, datatype which is unsupported in the issue... anywhere...

The description after obfuscation of objectname can even act as a key for case like `index on complex datatypes 'citext'`

Reply:
Agreed but based on what we had discussed earlier, the solution would be for us to define separate issues (index-not-supported-json, index-not-supported-citext, etc). 

Only in cases where that's not possible (for instance, extensions), we could somehow pass that information along using the TYPE itself, or a different field like ObjectName